cou/commands.py

```diff
@@ -185,8 +168,11 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
     )
     subcommand_common_opts_parser.add_argument(
         "--archive-batch-size",
-        help="Batch size for nova old database data archiving.\n"
-        "Decrease the batch size if performance issues are detected.\n(default: 1000)",
+        help=(
+            "Batch size for nova old database data archiving."
+            "\nDecrease the batch size if performance issues are detected.\n(default: 1000)"
+            "\nThis option requires --archive."
```

Comment:
Could you also add the logic to enforce this, copying the logic for purge/purge-before-date?

Reply:
(Alternatively, that could be done in another PR, but perhaps the changes to --archive help message should be out of scope here then.)

Reply:
Ah, sorry, this was accidentally added. This option does not required `--archive`